apiml/src/main/java/org/zowe/apiml/controller/IncorrectPassTicketRequestBodyException.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+
+package org.zowe.apiml.controller;
+
+public class IncorrectPassTicketRequestBodyException extends RuntimeException {
```

Comment:
SerialUid and message?

Reply:
added servialuid, message is not needed in this case, exception is self contained, no parameters in the original one